clinvar_parser/clinvar-parser/src/main/java/uk/ac/ebi/eva/clinvar/ClinvarJsonSerializer.java

```diff
@@ -0,0 +1,56 @@
+package uk.ac.ebi.eva.clinvar;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import uk.ac.ebi.eva.clinvar.model.ClinvarSet;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.Callable;
+
+public class ClinvarJsonSerializer implements Callable<Integer> {
+
+    private final ObjectWriter jsonObjectWriter;
+
+    private final BufferedWriter bw;
+
+    private ArrayBlockingQueue<ClinvarSet> clinvarSetsQueue;
+
+    public ClinvarJsonSerializer(ArrayBlockingQueue<ClinvarSet> clinvarSetsQueue, BufferedWriter bw) throws IOException {
```

Comment:
Naming the first argument `inputQueue` or `outputQueue` like in `ClinvarSetTransformer` would make its purpose more clear.